.github/CODEOWNERS

```diff
@@ -4,5 +4,4 @@
 # most precedence.
 
 # Primary repo maintainers
-*       @aaronc @alexanderbez
-
+*       @cosmos/sdk-core-dev
```

Comment:
Give GH time for this error to go away, as I just have write access to the team for the SDK.